std/algorithm/comparison.d

```diff
@@ -475,10 +475,19 @@ unittest
                            )());
 }
 
-/**
-Returns $(D val), if it is between $(D lower) and $(D upper).
-Otherwise returns the nearest of the two. Equivalent to $(D max(lower,
-min(upper,val))).
+/** Clamps a value into bounds.
+
+This functions is equivalent to $(D max(lower, min(upper,val))).
+
+Params:
+    val = The value to clamp.
+    lower = The lower_ bound of the clamp.
+    upper = The upper_ bound of the clamp.
```

Comment:
Sorry, actually there was a mistake. It should be `_lower` and `_upper` to suppress highlighting. Putting the underscore at the end actually makes the underscore show up in the HTML, which is wrong.